ebu_tt_live/bindings/validation/base.py

```diff
@@ -145,6 +147,20 @@ def _semantic_deconflicted_ids(self, attr_name, dataset):
                 new_elem_ids.append(new_elem.id)
             return new_elem_ids
 
+    def get_attribute_value(self, att_name):
+        """
+        This function is a handy extension that allows us to easily look up attribute values regardless whether they
+        are local or namespaced attribute names. I did not find its equivalent in PyXB.
```

Comment:
As it happens the guy has lots on his plate so unless I am PR-ing back to him a decent implementation of this one it is pointless to do a ticket. I wrote him e-mail about an issue earlier and his response was basically: InsufficientTimeException :)